src/librustc_trans/intrinsic.rs

```diff
@@ -280,6 +280,11 @@ pub fn trans_intrinsic_call<'a, 'tcx>(bcx: &Builder<'a, 'tcx>,
                             let llfn = ccx.get_intrinsic(&format!("llvm.{}.i{}", name, width));
                             bcx.call(llfn, &[llargs[0], y], None)
                         }
+                        "ctlz_nonzero" | "cttz_nonzero" => {
+                            let y = C_bool(bcx.ccx, true);
+                            let llfn = ccx.get_intrinsic(&format!("llvm.{}.i{}", &name[..4], width));
```

Comment:
Yup; currently waiting on local double-check for the fix.  Thanks for the targeted comment!